version-bump/main.py

```diff
@@ -62,6 +69,17 @@ def update_xml(version, file):
         mytree.write(file, encoding="utf-8")
 
 
+# For updating Helm Charts - Chart.yaml version
+def update_yaml(version, file):
```

Comment:
for code consistency here, could we use either `file_path` instead of `file`?

Reply:
Done.